NetUtilities/NetUtilities.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;net45;net46;net47</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.0;netcoreapp2.1;netcoreapp2.2;net47</TargetFrameworks>
```

Comment:
C:\Program Files\dotnet\sdk\2.2.100-preview2-009404\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(199,5): error NU5026: The file 'C:\projects\netutilities\NetUtilities\bin\Any CPU\Release\net47\NetUtilities.dll' to be packed was not found on disk. [C:\projects\netutilities\NetUtilities\NetUtilities.csproj]